### PR TITLE
feat: Add the chatbot-java recipe

### PR DESCRIPTION
**Pull request by jmartisk**
### What does this PR do?

Add the `chatbot-java` recipe (created added via https://github.com/containers/ai-lab-recipes/pull/809) to the default recipe catalog

### CODE REVIEW

File: packages/backend/src/assets/ai.json [hunk 1]
```diff
@@ -69,6 +69,23 @@
       ],
       "backend": "llama-cpp"
     },
+    {
+      "id": "chatbot-java",
+      "description": "This is a Java-based chat demo application.",
+      "name": "Java-based ChatBot",
+      "repository": "https://github.com/containers/ai-lab-recipes",
+      "ref": "main",
```
Comment (jmartisk) [inline packages/backend/src/assets/ai.json:77]: Not sure what to put here until it's part of a release, so I used `main` for now.

Comment (feloy) [reply packages/backend/src/assets/ai.json:77]: We would prefer to have a fixed ref here. You can use the full commit ID of the current HEAD for this

Comment (jmartisk) [reply packages/backend/src/assets/ai.json]: Ok @feloy, done

File: packages/backend/src/assets/ai.json [hunk 2]
```diff
@@ -69,6 +69,23 @@
       ],
       "backend": "llama-cpp"
     },
+    {
+      "id": "chatbot-java",
```
Comment (feloy) [inline packages/backend/src/assets/ai.json:73]: perhaps do we want to specify it is quarkus-based, as we will probably add other java frameworks

**Review by feloy — COMMENTED** [review]
Tested, works as expected.
Thanks for this contribution.

File: packages/backend/src/assets/ai.json (hunk 2 above)
Comment (maxandersen) [reply packages/backend/src/assets/ai.json:73]: I personally would prefer simple names to not crowd the UI in podman desktop catalog view
yeah I was wondering same thing; but as the list in here just says "chatbot" without mentioning python nor its streamlit based .... so the biggest diff which in this case is "java" seemed good balance.

maybe `chatbot-java-quarkus`, `chatbot-python-streamlit`, `chatbot-python-flask` etc. is a good pattern to apply?

Comment (benoitf) [inline packages/backend/src/assets/ai.json:73]:
```suggestion
      "id": "chatbot-java-quarkus",
```

so it'll match suggested `exampleName-language-framework/platform`

**Review by benoitf — APPROVED** [review]

**Review by feloy — APPROVED** [review]
LGTM

File: packages/backend/src/assets/ai.json (hunk 2 above)
Comment (jmartisk) [reply packages/backend/src/assets/ai.json:73]: @rhatdan / @jeffmaury agree with renaming this recipe (I'd first change it in https://github.com/containers/ai-lab-recipes for consistency)?

Comment (benoitf) [reply packages/backend/src/assets/ai.json:73]: yes please change first into ai-lab-recipes for consistency

**Comment by jmartisk** [comment]
I've sent https://github.com/containers/ai-lab-recipes/pull/813 for the rename, so merging this one should wait for that one (I'll also have to update the commit hash after https://github.com/containers/ai-lab-recipes/pull/813 is merged)

**Comment by jmartisk** [comment]
Ok, renamed it and updated the git ref, should be all good now

**Comment by benoitf** [comment]
I launched the CI workflows

**Comment by benoitf** [comment]
thanks @jmartisk

**Pull request merged**